### PR TITLE
Functional thumbs up/down component 

### PR DESCRIPTION
**Pull request by albandum**
## Description
- Make the thumbs up/down component functional
- In this PR, it doesn't have the author displayed yet - will do in a future PR

https://github.com/user-attachments/assets/3b30209d-6575-436e-9d08-fc6ce87a4300

### CODE REVIEW

File: sparkle/src/components/ConversationMessageActions.tsx [hunk 1]
```diff
@@ -24,20 +24,21 @@ import { cn } from "@sparkle/lib/utils";
...
-      <ConversationMessageThumbsSelector
+      <ConversationMessageFeedbackSelector
         key="thumbs-selector"
-        onSubmitThumb={messageThumb.onSubmitThumb}
-        isSubmittingThumb={messageThumb.isSubmittingThumb}
+        feedback={messageFeedback.feedback}
+        onSubmitThumb={messageFeedback.onSubmitThumb}
+        isSubmittingThumb={messageFeedback.isSubmittingThumb}
       />
     );
```
Comment (tdraier) [inline sparkle/src/components/ConversationMessageActions.tsx:43]: It's not directly related to your PR (it's actually the code I moved from front to here) but this code is not ok and it give very strange results in storybook (surprising it works in the app) - new buttons are added on every render - and it's modifying a prop, which is also not always a good idea. Couldn't we just export the 2 components ConversationMessageThumbsSelector and ConversationMessageEmojiSelector and let the user pass the buttons he needs ?

**Review by tdraier — COMMENTED** [review]
LGTM but I think we should change the way the action buttons are passed and get rid of the ConversationMessageActions component, that would make everything much easier to use

File: sparkle/src/components/ConversationMessageActions.tsx (hunk 1 above)
Comment (albandum) [reply sparkle/src/components/ConversationMessageActions.tsx:43]: Agreed on storybook behavior I found it weird too.
Not sure what you mean by "let the user pass the buttons he needs"?

Comment (albandum) [reply sparkle/src/components/ConversationMessageActions.tsx]: Moved the feedbackSelector out of ConversationMessageActions - revamping the rest should belong to another cleanup PR

Comment (tdraier) [reply sparkle/src/components/ConversationMessageActions.tsx]: do something like 
```
<ConversationMessage messages={[
  <ConversationMessageEmojiSelector ../>,
  <ConversationMessageFeedbackSelector ../>
]} />
```

**Review by tdraier — APPROVED** [review]
lgtm ! 🔥

**Comment by albandum** [comment]
> wait - where is the ConversationMessageEmojiSelector gone ?

We're removing - replacing with the thumbs

**Pull request merged**